cluster/saltbase/salt/calico/calico-policy-controller.manifest

```diff
@@ -1,21 +1,41 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: calico-policy-controller
+  name: policy-controller
   namespace: kube-system
   labels:
+    version: "v0.2.0"
```

Comment:
Exactly what labels did he want then?

I'll do the same for this and the calico-node manifest.